packages/shared/src/components/post/common/YouTubeVideo.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+
+interface YouTubeVideoProps {
+  title: string;
+  videoId: string;
+  className?: string;
+}
+
+export function YouTubeVideo({
```

Comment:
This already existed right?

Reply:
Ah, right. Should remove the TODO. This is where the article modal components are used.

Reply:
Not quite sure, do you know the exact component?

Reply:
Thanks! Removed this file now 🚢